stdlib/unittest/case.pyi

```diff
@@ -27,6 +27,23 @@ DIFF_OMITTED: str
 class _BaseTestCaseContext:
     def __init__(self, test_case: TestCase) -> None: ...
```

Comment:
```suggestion
class _BaseTestCaseContext:
    test_case: TestCase
    def __init__(self, test_case: TestCase) -> None: ...
```

Reply:
good catch - I can remove `test_case: TestCase` from some of the child classes with that too.